src/renderer/components/dock/create-resource.tsx

```diff
@@ -88,7 +89,10 @@ export class CreateResource extends React.Component<Props> {
 
   onSelectTemplate = (item: SelectOption) => {
     this.currentTemplates.set(this.tabId, item);
-    fs.readFile(item.value,"utf8").then(v => createResourceStore.setData(this.tabId,v));
+    fs.readFile(item.value,"utf8").then(v => {
+      createResourceStore.setData(this.tabId,v);
+      monacoModelsStore.getModel(this.tabId).setValue(v ?? "");
```

Comment:
Do we need both of these stores? Unless they do different tasks it seems like it is making more work for ourselves by having to keep them in sync.

Reply:
Basically, monacoModelsStore is for managing monaco models, these entities are responsible for storing virtual in-memory "profiles" for the tabs. I tried to move all logic in one object, but, as I see, it requires more global refactoring. 

I can remove storing functions from monacoModelsStore (and rename this to something like monacoModelManager) and work with this within runtime, w/o (de)serialization of urls and languages.

Reply:
If it is just for in memory remembering anyway. I think that just a model manager would be good.

Reply:
fixed